package.json

```diff
@@ -58,9 +58,6 @@
     "pretty-quick": "^3.1.0",
     "remark": "^12.0.1"
   },
-  "peerDependencies": {
-    "gatsby": "^2.20.0"
-  },
```

Comment:
Technically you don't *have* to use gatsby to use this project.

I think the risk of someone mistakenly installing this without gatsby is extremely low, so I don't think there's much of a concern here.